syntax = "proto3";

package DO_and_QU;

// The greeting service definition.
service DOQU_Greeter {
  rpc PublicKey (PublicKeyRequest) returns (ParameterReply) {}
}

// QU向DO发送公钥
message PublicKeyRequest {
  string publicKey = 1;
}

// DO向QU发送加密的希尔伯特曲线参数（vector<string>）
message ParameterReply {
  repeated bytes parameter = 1;
  string conversionKey = 2;
  string publicKey = 3;
}